tests/test_service_account.py

```diff
@@ -601,19 +601,23 @@ def test_authorize_stale_token(self, utcnow):
 
     @mock.patch('oauth2client.client._UTCNOW')
     def test_authorize_401(self, utcnow):
+        """Test that a 401 triggers a refresh of the access token."""
```

Comment:
Don't use docstrings in unit tests, it can screw up the output. But maybe that only matters for `nose`, though people may still use `nose`